tensorflow_quantum/python/differentiators/linear_combination.py

```diff
@@ -111,7 +111,6 @@ def __init__(self, weights, perturbations):
         self.n_non_zero_perturbations = tf.gather(
             tf.shape(self.non_zero_perturbations), 0)
 
-    @tf.function
```

Comment:
Wouldn't we want to keep this around ?

Reply:
Added back, I had removed during debugging and forgot